frontend/app/api/concepts.json

```diff
@@ -59,10 +59,15 @@
             {
               "id": "budget",
               "label": "Budget",
-              "type": "INTEGER_RANGE",
+              "type": "MONEY_RANGE",
               "unit": "1K USD",
               "description": "The movie's total budget",
-              "min": 1
+              "min": 1,
+              "defaultValue": {
```

Comment:
I still think we should rather demo this in the example stored query, instead of always settings this default value. 
What movie has a budget smaller than 44,55 Euros? :wink:
Also, the value for `exact` has to be removed.